.travis.yml

```diff
@@ -11,6 +11,8 @@ env:
     TASKS="bench:compile"
   - TEST_DIR=offline-regression-report
     TASKS="bench:compile"
+  - TEST_DIR=invocation-count-measurers
+    TASKS="bench:compile"
```

Comment:
Whatever is more convenient for you. Since it will increase the running time, might be better to do a separate PR.